plugins/modules/purefa_pod.py

```diff
@@ -625,7 +625,9 @@ def delete_pod(module, array):
     changed = True
     if not module.check_mode:
         res = array.patch_pods(
-            names=[module.params["name"]], pod=PodPatch(destroyed=True)
+            names=[module.params["name"]],
+            pod=PodPatch(destroyed=True),
+            eradicate_contents=module.params["delete_contents"],
```

Comment:
For pod deletion to work fine, should change the eradicate_contents parameter to destroy_contents.

Reply:
Agreed